api/main_endpoints/routes/officerManager.js

```diff
@@ -22,7 +22,7 @@ const membershipState = require('../../util/constants').MEMBERSHIP_STATE;
 router.post('/submit', (req, res) => {
   if (!checkIfTokenSent(req)) {
     return res.sendStatus(FORBIDDEN);
-  } else if (!checkIfTokenValid(req, membershipState.OFFICER)) {
+  } else if (!checkIfTokenValid(req)) {
```

Comment:
is there a reason why the `membershipState.OFFICER` param was removed?

Reply:
I remember debugging this file and I may have accidentally forgotten to put it back. I put it back now.